### ISSUE TITLE
Triage access for sophiagustafson

### ISSUE THREAD

**Issue by sophiagustafson** [defund12/defund12.org#1085]
# Triage access

👋 I am a defund12.org collaborator requesting triage permissions so I can assist in changing new-request labels to either closed or ready-for-review.

**Comment by bingbongle** [comment]
Hey! Can you provide some examples here of contributions you've made to the project. Sorry for the hold up but we're slowly rolling out access and prioritizing early on / high volume contributors!

**Comment by sophiagustafson** [comment]
Yeah! I have written the email templates for Irving, Omaha, Lincoln, and Anchorage.
